Intg 423 (#1)
invitation-homes/mulesoft-build-action: README.md

# Mulesoft Build Action

> Mulesoft build action compiles/build Mulesoft maven project, create a release on GitHub, upload and associate build artifact with a produced release.

Reviewer: ```suggestion
A GitHub Action that uses maven to compile a jar from a Mulesoft project, creates a release on GitHub, and uploads the jar to that release.
```

## Usage
Add the following steps in your workflow:

```
    - uses: invitation-homes/mulesoft-build-action@main
        env:

>           github_token:  "GITHUB_TOKEN"

Reviewer: After looking at some of the official actions, it seems like the convention is for environment variable names to be upper case, e.g.
```
GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
I'll grant you that there is plenty of variety out there, but upper case seems to be the most prevalent. That's how we've named our secrets too, so it's probably best to be consistent with upper case. We should do that here as well as in the JavaScript code (which is mostly the case already).

>           mulesoft_nexus_user: "MULESOFT Enterprise Repo. User"

Reviewer: I would go ahead and include the actual `${{ secrets.MULESOFT_NEXUS_USER }}` or whatever on the right side. Make it easy for our other developers to just cut & paste the code into their workflow.
